### DISCUSSION TITLE
How to get a rotated version of a vector

### DISCUSSION THREAD

**Discussion by Drumstickz64** [bevyengine/bevy#3350] [Q&A] [answered]
How do I get a rotated version of a vector (like `Vec3`)? Is there a function or method like `Vector3.rotated` from godot, in bevy?

**Comment by DJMcNab** [comment, marked as answer]
A quick search of the glam docs brings up [`Quat::mul_vec3`](https://docs.rs/glam/0.20.1/glam/f32/struct.Quat.html#method.mul_vec3), which you should be able to use to rotate your vector

**Reply by Drumstickz64** [reply]
Ah when I saw this method I thought it rotated the quaternion, not the vector. Thanks a bunch!
